perf/benchmark/runner/runner.py

```diff
@@ -42,17 +42,21 @@ def pod_info(filterstr="", namespace=NAMESPACE, multi_ok=True):
     cmd = "kubectl -n {namespace} get pod {filterstr}  -o json".format(
         namespace=namespace, filterstr=filterstr)
     op = getoutput(cmd)
-    o = json.loads(op)
-    items = o['items']
-
-    if not multi_ok and len(items) > 1:
-        raise Exception("more than one found " + op)
+    o = None
+    try:
+        o = json.loads(op)
+    except ValueError:
+        print("Decoding pod spec JSON has failed!")
```

Comment:
probably better to figure out why it failed other than just print the errors here, seems like we got extra json data record?

Reply:
@richardwxn yes, something like that, since I could not reproduce this error on my local, so I put this fix on hold...

Reply:
not sure if this by chance or for some specific reasons..